manufacturing/oxygen/README.md

```diff
@@ -0,0 +1,147 @@
+| 3    |        1 | RespiraWorks  |                     |              | [B][3rw]       | 1/4"ID x 3/4"ID reducer |
+
+[1mcmc]: https://www.mcmaster.com/5894K34
+[2mcmc]: https://www.mcmaster.com/5121K451
+[3rw]: #tubing-reducer
+
+### Tubing reducer
+
+The first few components on the oxygen line will use 1/4"ID tubing, before transitioning to the 3/4" used throughout
+the rest of the ventilator assembly. Unfortunately, there is no readily available off-the-shelf reducer for interfacing
+these diameters. For this purpose, we have created a custom 3d-printable adapter.
+
+|  CAD rendering    |  3d printed   |
+:------------------:|:-----------------:|
+![](reducer/reducer_CAD.jpg)  |  ![](reducer/reducer_printed.jpg)  |
+
+The following files are provided:
+* [STL](reducer/Reducer%201-4in%20x%203-4in.stl) - exported as high resolution for 3d printing.
+* [IPT](reducer/Reducer%201-4in%20x%203-4in.ipt) - AutoDesk Inventor, original CAD with constraints.
+
+In a future iteration, the reducer may be integrated right into the venturi.
+
+## Check valve
+
+We are currently testing with this brass threaded check valve:
+
+https://www.mcmaster.com/7768K22/
+
+It interfaces with a through-wall straight connector that will be on the enclosed ventilator's front panel:
+
+https://www.mcmaster.com/50785K273/
+
+On the opposite end of the fitting, a 1/4"ID tubing adapter (item 3 in [materials list above](#materials)) should be
+attached. Together with this adapter, the oxygen input port should appear like this:
+
+|  Front    |  Back   |
+:------------------:|:-----------------:|
+![](images/through-fitting_1.jpg)  |  ![](images/through-fitting_2.jpg)  |
+
+It is not clear whether the check valve has the correct cracking pressure and other pneumatic properties for our system.
+
+Related issues are being tracked here:
+[#653](https://github.com/RespiraWorks/Ventilator/issues/653)
+[#646](https://github.com/RespiraWorks/Ventilator/issues/646)
+
+## Oxygen regulator
+
+We are currently testing with
+
+https://www.smcpneumatics.com/IR2010-N02B.html
+
+It requires 2x of 1/4"ID tubing adapter (item 3 in [materials list above](#materials)) to interface with tubing.
+
+## Proportional solenoid valve
+
+We are currently testing with:
+
+https://www.smcpneumatics.com/PVQ31-6G-40-01N.html
+
+It requires 2x of 1/4"ID tubing adapter (item 3 in [materials list above](#materials)) to interface with tubing.
+
+Depending on the version procured (NPT or BSPT), it may also required 2x of the following adapters:
+
+https://www.mcmaster.com/1786N249/
+
+In collaboration with Foundry-M in India, we are also testing with the following locally sourced solenoids:
+
+https://www.uflowvalve.com/solenoid-proportional-flow-control-valve/product/2-2-way-miniature-high-flow-proportional-valve
+
+https://www.uflowvalve.com/solenoid-proportional-flow-control-valve/product/2-2-way-miniature-low-flow-proportional-valve
+
+## Air flow sensor
```

Comment:
Might want to call this a gas flow sensor to avoid confusion

Reply:
Changed.